components/analytics/table_chart.tsx

```diff
@@ -1,43 +1,22 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 
 import Constants from 'utils/constants.jsx';
 
 export interface TableItem {
-    tip: string;
     name: string;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    value: any;
+    tip: string;
+    value: React.ReactNode;
 }
 export interface TableChartProps {
```

Comment:
we are using local type aliases for component props because they are used locally only, and actually, we don't need to export them (at least for now).

Reply:
Okay, I'll revert the `export`.